modules/gitops-inbuilt-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-inbuilt-permissions-for-cluster-config_{context}"]
+= In-built permissions for cluster configuration
+
+The default Argo CD instance has some default permissions to manage specific cluster-scoped resources which include platform operators, optional OLM operators and user management. 
+
+[NOTE]
+====
+Argo CD will not have cluster-admin permissions.
+====
+
+The following table lists the permissions for the Argo CD instance:
+
+|===
```

Comment:
Suggestion: In general, I would remove the period at the end of the statements in the right-hand column in the table, as they are not stand-alone sentences.